accelerator/config.c

```diff
@@ -5,10 +5,74 @@
 
 static logger_id_t logger_id;
 
-status_t ta_config_init(ta_config_t* const info, iota_config_t* const tangle,
-                        iota_client_service_t* const service) {
+struct option* cli_build_options() {
```

Comment:
maybe we can use `typedef` to make the decalration more clear?

Reply:
We can pass `ta_core_t` instead.

Reply:
It's data structure defined in <getopt.h>, I think we just keep it this way.